### PR TITLE
[mqtt.espmilighthub] Update README.md

### PR DESCRIPTION
**Pull request by hutcheonk**
Add "MQTT Client Status Messages Mode setting" to ESP Config..

### CODE REVIEW

**Comment by Skinah** [comment]
LGTM.... Thanks for PR,  hope you make many more as all contribs are welcome.

**Review by clinique — APPROVED** [review]

File: bundles/org.openhab.binding.mqtt.espmilighthub/README.md [hunk 1]
```diff
@@ -45,6 +45,9 @@ Leave this blank.
 **MQTT Client Status Topic:**
 `milight/status`
 
+**MQTT Client Status Messages Mode:**
+Simple.
```
Comment (J-N-K) [inline bundles/org.openhab.binding.mqtt.espmilighthub/README.md:49]: Maybe it's so simple that I don't understand it. What exactly needs to be set? Is "Simple" the option you have to choose?

**Review by Skinah — APPROVED** [review]
LGTM

File: bundles/org.openhab.binding.mqtt.espmilighthub/README.md (hunk 1 above)
Comment (Skinah) [reply bundles/org.openhab.binding.mqtt.espmilighthub/README.md:49]: `Simple` is the option you select from the box labelled **MQTT Client Status Messages Mode:** It makes sense when you look at the GUI of the device and the rest of the setup is  formatted the same way.

Comment (lolodomo) [reply bundles/org.openhab.binding.mqtt.espmilighthub/README.md:49]: @J-N-K : is it ok for you now ?

Comment (Skinah) [reply bundles/org.openhab.binding.mqtt.espmilighthub/README.md:49]: @J-N-K  as above are you happy?

**Comment by lolodomo** [comment]
As you have no feedback from @J-N-K since several weeks and the official maintainer of this binding is fine with the change, let's merge this PR.

**Pull request merged**